src/components/AccountLoginForm.jsx

```diff
@@ -27,6 +27,10 @@ const AccountLoginForm = ({ t, isOAuth, oAuthTerminated, fields, error, dirty, s
             : ''
           switch (fields[name].type) {
             case 'password':
+              if (isUnloading) {
+                fields[name]['value'] = ''
```

Comment:
By changing directly this property, we are mutating or data and it could have undesired side-effects. I suggest to pass directly a new object to the `Field` component created with the spread operator, with something like:
```jsx
switch (fields[name].type) {
  case 'password':
    return <div>
      {description}
      <PasswordField
           // [..]
          {...fields[name]}, ...{ value: isUnloading ? '' : fields[name].value  } />
     </div>
```